server/routes/profile.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  createProfile,
+  getProfile,
+  patchProfile,
+  getAllProfiles,
+} = require("../controllers/profile");
+
+router.route("/profile/create").post(protect, createProfile);
```

Comment:
```suggestion
router.route("/").post(protect, createProfile);
```
The route names can be simplified as the HTTP verbs signify the purpose of the route. Ditto throughout!